src/base/bittorrent/session.cpp

```diff
@@ -1313,6 +1314,9 @@ void Session::configure(libtorrent::settings_pack &settingsPack)
 
     settingsPack.set_int(libt::settings_pack::aio_threads, asyncIOThreads());
 
+    const int checkingMemUsageSize = checkingMemUsage() * 64;
```

Comment:
why is this multiplied by 64 here?

Reply:
Because the user specifies the value in MiBs in the GUI and this setting is specified in number of 16 kiB blocks for libtorrent.
So, when the user selects x MiB in the GUI, the corresponding number of 16 kiB blocks will be (x * 1024) / 16 = x * 64, which will be the value passed onto libtorrent.